docker/Dockerfile

```diff
@@ -36,6 +36,10 @@ ENV DOWNLOAD=${DOWNLOAD}
 
 MAINTAINER FIWARE IoTAgent Team. Telefónica I+D
 
+RUN apt-get update \
+	&& apt-get install -y --no-install-recommends ca-certificates curl wget \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
I'd suggest to add the new commands to the main building RUN at L64

Not sure if a matter of style or efficiency  (not sure, but each RUN statement is a diferent image layer) but that's the way I understand we typically used in this and other IOTAs. repository.

Reply:
Fixed in 8f0be3b